Makefile

```diff
@@ -27,6 +27,9 @@ RELEASE_VERSION?=$(shell git describe --tags --match "v*")
 RELEASE_IMAGE:=kube-scheduler:$(RELEASE_VERSION)
 RELEASE_CONTROLLER_IMAGE:=controller:$(RELEASE_VERSION)
 
+# VERSION is the scheduler's version
+VERSION=$(shell git describe --tags --match 'v[0-9]*' --always)
```

Comment:
FYI ... this will not work in the automated container image build environment that produces images for k8s.gcr.io. The `VERSION` variable will end up being an empty string.

This variable will need to be set in the [cloudbuild.yaml](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/cloudbuild.yaml) file just like `RELEASE_VERSION` if you want it to work for the automated container image builds.

Reply:
@seanmalloy do you happen to know the mechanics used in default scheduler? i.e., does `--version` in default scheduler  output correct version info in an automatic container build?

Reply:
@Huang-Wei I'm not familiar with the how this works in the default scheduler.

Pulling the latest scheduler-plugins image from staging registry give this error.
```
docker run gcr.io/k8s-staging-scheduler-plugins/kube-scheduler:v20201009-v0.18.800-46-g939c1c0 /bin/kube-scheduler --version
panic: version string "" doesn't match expected regular expression: "^v(\d+\.\d+\.\d+)"

goroutine 1 [running]:
k8s.io/component-base/metrics.parseVersion(0x0, 0x0, 0x0, 0x0, 0x1eaa46e, 0x0, 0x1caa924, 0xb, 0x0, 0x0, ...)
        /go/src/sigs.k8s.io/scheduler-plugins/vendor/k8s.io/component-base/metrics/version_parser.go:47 +0x26b
k8s.io/component-base/metrics.newKubeRegistry(0x0, 0x0, 0x0, 0x0, 0x1eaa46e, 0x0, 0x1caa924, 0xb, 0x0, 0x0, ...)
        /go/src/sigs.k8s.io/scheduler-plugins/vendor/k8s.io/component-base/metrics/registry.go:312 +0xfb
k8s.io/component-base/metrics.NewKubeRegistry(0x9, 0xc000204280)
        /go/src/sigs.k8s.io/scheduler-plugins/vendor/k8s.io/component-base/metrics/registry.go:327 +0x7f
k8s.io/component-base/metrics/legacyregistry.init()
        /go/src/sigs.k8s.io/scheduler-plugins/vendor/k8s.io/component-base/metrics/legacyregistry/registry.go:29 +0x26
```

I can also say that it is also broken in the descheduler. Not that "Major" and "Minor" versions empty.
```
docker run k8s.gcr.io/descheduler/descheduler:v0.19.0 /bin/descheduler version
Descheduler version {Major: Minor: GitCommit: GitVersion: BuildDate:2020-09-01T16:43:23+0000 GoVersion:go1.15 Compiler:gc Platform:linux/amd64}
```

Reply:
@Huang-Wei and @yuanchen8911 I think I know how tot fix this for the scheduler-plugins. Let me know if you want me to go ahead and submit a PR. Thanks!

Reply:
You rocks! Please go ahead to fix this.

Reply:
See proposed fix #76.